src/compositor/lipstickcompositor.cpp

```diff
@@ -704,7 +707,9 @@ void LipstickCompositor::setUpdatesEnabled(bool enabled)
             if (QWindow::handle()) {
                 QGuiApplication::platformNativeInterface()->nativeResourceForIntegration("DisplayOff");
             }
+            m_fakeRepaintLoop.start();
```

Comment:
Right, we could listen for commit()s and trigger a fake repaint after some delay.